app/src/main/res/layout/activity_signup.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    tools:context=".features.signup.SignupActivity">
+
+    <EditText
+        android:id="@+id/signup_firstName"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center"
+        android:layout_margin="15dp"
```

Comment:
If we're using 15dp a lot, can we declare it in a dimension file?

Reply:
I was holding off changing dimens file until we have designs since we will be changing all these anyways but I can if you want to,